r2-core/src/main/java/com/linkedin/r2/transport/http/client/RateLimiter.java

```diff
@@ -57,6 +57,11 @@
    */
   Collection<Task> cancelPendingTasks();
 
+  default int numberOfPendingTasks()
+  {
+    return -1;
+  }
+
```

Comment:
There is not explicit need for having this other than testing. Our usual stance is to add methods to interfaces only if really needed. Adding it in the interface means extra support in the future for any ratelimiter.

I'd rather remove this and make the method @VisibleForTesting

Reply:
Sure, we do not have VisibleForTesting anywhere in pegasus. Need to have additional external dependency  for that. To make things simpler I will have this method part of ExponentialRateLimitor alone and not part of the interface.